packages/fluent-theme/src/components/activity/ActivityDecorator.module.css

```diff
@@ -0,0 +1,301 @@
+
+  /* TODO: remove when we get decorators in and can place header direclty to the bubble*/
+  &:not(:has(:global(.webchat__bubble--from-user))) :global(.webchat__stacked-layout .webchat__bubble .webchat__text-content) {
+    padding-block: calc(var(--webchat-spacingVerticalS) + 20px) var(--webchat-spacingVerticalXXS);
+  }
+  &:not(:has(:global(.webchat__bubble--from-user))) :global(.webchat__stacked-layout .webchat__bubble) {
+    margin-block-start: -24px;
+    margin-inline-start: -4px;
+  }
+}
+
+/* Transcript activity non-empty */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__basic-transcript__activity-body:not(:empty)) {
+  padding-block-end: var(--webchat-spacingVerticalXL);
+}
+
+/* Stacked layout */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__stacked-layout) {
+  display: flex;
+  flex-flow: column nowrap;
+}
+
+/* Stacked layout which has message bubble */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__stacked-layout .webchat__stacked-layout__content:has(.webchat__bubble)) {
+  overflow: visible;
+}
+
+/* Message status */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__stacked-layout .webchat__stacked-layout__status) {
+  font-size: var(--webchat__font-size--small);
+  line-height: var(--webchat__line-height--small);
+}
+
+/* Message bubble */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__stacked-layout .webchat__bubble) {
+  overflow: visible;
+  max-width: var(--webchat__bubble--max-width);
+}
+
+/* Message bubble content */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__stacked-layout .webchat__bubble .webchat__bubble__content) {
+  background-color: var(--webchat__bubble--background-color);
+  border-radius: var(--webchat__bubble--border-radius);
+  border-width: 0;
+  box-sizing: border-box;
+  color: var(--webchat-colorNeutralForeground1);
+  min-height: var(--webchat__bubble--min-height);
+}
+
+/* Message bubble text content */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__stacked-layout .webchat__bubble .webchat__text-content) {
+  font-size: var(--webchat__font-size--medium);
+  line-height: var(--webchat__line-height--medium);
+  min-height: auto;
+  padding-block: var(--webchat__bubble--block-padding);
+  padding-inline: var(--webchat__bubble--inline-padding);
+}
+
+/* Message bubble content pseudo */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__stacked-layout .webchat__bubble .webchat__bubble__content)::before {
+  content: '';
+  border-radius: inherit;
+  box-shadow: var(--webchat__bubble--box-shadow);
+  inset: 0;
+  isolation: isolate;
+  pointer-events: none;
+  position: absolute;
+}
+
+/* Markdown links and citation links */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__render-markdown) a[href]:not(:global(.webchat__render-markdown__pure-identifier)) {
+  color: var(--webchat-colorBrandForegroundLink);
+  text-decoration-color: transparent;
+
+  &:target {
```

Comment:
Should we keep this if it doesn't work across shadow DOM?

https://developer.mozilla.org/en-US/docs/Web/CSS/:target

Reply:
Who knows maybe it'll get eventually fixed. So far Web Chat doesn't support shadow DOM, I'd leave it for now